src/index.js

```diff
@@ -245,6 +258,12 @@ class Masq {
         if (!json.userAppDEK) {
           throw new MasqError(ERRORS.WRONG_MESSAGE, 'User app dataEncryptionKey (userAppDEK) not found in "masqAccessGranted" message')
         }
+        if (!json.username) {
+          throw new MasqError(ERRORS.WRONG_MESSAGE, 'Username not found in \'authorized\' message')
```

Comment:
message 'masqAccessGranted' not 'authorized'

Reply:
ok done